### ISSUE TITLE
six.text_type('') takes two passes to resolve

### ISSUE THREAD

**Issue by hugovk** [asottile/pyupgrade#248]
For example:

```console
$ pip freeze | grep pyupgrade
pyupgrade==1.26.1
$ cat 1.py
six.text_type('')
$ pyupgrade 1.py --py3-plus
Rewriting 1.py
$ cat 1.py
str('')
$ pyupgrade 1.py --py3-plus
Rewriting 1.py
$ cat 1.py
''
$ pyupgrade 1.py --py3-plus
$
```

Perhaps rewrite directly to `''`?

**Comment by asottile** [comment]
I believe the code hint is here: https://github.com/asottile/pyupgrade/blob/465effb0a65f3cd05d8b8201be4c8b899941b6e1/pyupgrade.py#L1476

**Comment by asottile** [comment]
released in v1.26.2 -- thanks for the issue!

**Comment by hugovk** [comment]
And thanks for the update!
